ingestion-beam/src/main/java/com/mozilla/telemetry/decoder/ParseUri.java

```diff
@@ -245,10 +245,10 @@ private static class InvalidIntegerException extends InvalidUriException {
     public static final String FUNNELCAKE = "funnelcake";
 
     public static final Pattern PING_VERSION_PATTERN = Pattern
-        .compile(String.format("^v(?<%s>[6-9])(-(?<%s>\\d+))?$", VERSION, FUNNELCAKE));
+        .compile(String.format("^v(?<%s>[6-10])(-(?<%s>\\d+))?$", VERSION, FUNNELCAKE));
```

Comment:
I'm not sure why the error message in CI is cryptic but the issue is this regex is invalid.  Something like this should work to get numbers [6, 19]:
```suggestion
        .compile(String.format("^v(?<%s>[6-9]|1[0-9])(-(?<%s>\\d+))?$", VERSION, FUNNELCAKE));
```
I would also recommend adding a v10 test case to https://github.com/mozilla/gcp-ingestion/blob/fb7e9ed9e891e3e2320d85e05d7c1a1aedb57780/ingestion-beam/src/test/java/com/mozilla/telemetry/decoder/ParseUriTest.java#L321
This might be tricky to do if you don't have the environment setup so I can help with this if you need